### ISSUE TITLE
Mac Address

### ISSUE THREAD

**Issue by gh0st19** [mvasilchuk/webproxy#1]
Does each emulator have to be run using different mac addresses when running under the proxy?

**Comment by mvasilchuk** [comment]
No. They all have to use the same MAC, if you want to spoof device id. Read http://wiki.stbemu.com/index.php/WebProxy for more info (sorry, I forgot to translate it into English, so use Google translator to read it if you don't know Russian).

**Comment by gh0st19** [comment]
Thank you for getting back to me. Do you need to run a separate instance of
the proxy for each emulator? I seem to be getting the stb damaged message
after loading. I connected to my mag254 and the id populates correctly in
the ini.

Thank you

**Comment by mvasilchuk** [comment]
You need to run only one instance of proxy. Then just put proxy's ip address into the emulator's profile instead of address of a portal (but keep path, i.e. /stalker_portal/c/).

**Comment by gh0st19** [comment]
I must be doing something incorrectly the portal will start to load but it
will never make it to the home screen. It seems to work fine on Kodi but
the emu will just hang at startup. If I turn off allow emulator detection
then it will tell me my stb is damaged. Is there additional settings I have
to change besides the portal url and mac?

**Comment by gh0st19** [comment]
Is there a way to enter in a username or password into the webproxy.ini?

Regards,

**Comment by mvasilchuk** [comment]
Hi,

If you mean the login/password to access a portal, then the answer is no.
You have to write them into fields in portal and all devices that are
connected to a webproxy should work.
If you mean basic HTTP authorization, then you have to write login/password
in portal's url (e.g. url = http://login:password@iptv.provider.com
http://iptv.provider.com/)

2016-01-31 19:39 GMT+02:00 gh0st19 notifications@github.com:

> Is there a way to enter in a username or password into the webproxy.ini?
> 
> Regards,
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/mvasilchuk/webproxy/issues/1#issuecomment-177559512.

## 

Best regards,
Maxim Vasilchuk
